addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/dialogs/ZestRecordScriptDialog.java

```diff
@@ -146,6 +147,8 @@ private List<String> getScriptTypes() {
                 list.add(Constant.messages.getString(st.getI18nKey()));
             }
         }
+        // Alphabetic order best, and it just so happens Auth scripts will appear at the top
+        Collections.sort(list);
```

Comment:
The changes in this class seem unrelated.

Reply:
🤷‍♂️ from Simon's branch.

Will roll back.

Reply:
In that case I'll def add them in another PR as we do need them 😁